packages/perseus/src/widgets/__tests__/explanation.cypress.ts

```diff
@@ -0,0 +1,75 @@
+import renderQuestion from "../../../../../testing/render-question-with-cypress";
```

Comment:
**question**: What was the reason for needing Cypress tests here?  FWIW: we prefer @testing-library (RTL) tests over Cypress except in cases where we can't test a component with RTL.  Cypress tests are much slower and clumsier to write/work with so that's why.

Reply:
RTL is unable to test the styling that is applied to a given element. These tests are intended to verify that the styling that affects tab order do not regress. I will add comments to give the reasoning.